cockatrice/src/main.cpp

```diff
@@ -97,6 +100,19 @@ bool settingsValid()
         !settingsCache->getPicsPath().isEmpty();
 }
 
+void generateClientID()
```

Comment:
It would be enough to test that it generates the same id when called twice.

On Fri, Aug 7, 2015 at 8:59 AM woogerboy21 <notifications@github.com> wrote:

> In cockatrice/src/main.cpp
> <https://github.com/Cockatrice/Cockatrice/pull/1340#discussion_r36515795>:
>
> > @@ -97,6 +100,19 @@ bool settingsValid()
> >          !settingsCache->getPicsPath().isEmpty();
> >  }
> >
> > +void generateClientID()
>
> Could you also explain to me how you would call the function and nothing
> else when the function is inside a file that includes/contains the 2
> required components (network and settings)? I can understand what your
> saying to a point and fully intend on fixing it, but im just curious if
> your stretching things on this one a bit.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Cockatrice/Cockatrice/pull/1340/files#r36515795>.
>